tests/test_plot.py

```diff
@@ -726,3 +726,9 @@ def test_quantile_plot_interval_tuple(q1, q2):
     _q1, _q2 = quantile_plot_interval(q=(q1, q2))
     assert _q1 == q1
     assert _q2 == q2
+
+
+def test_basic_cmap_reversible():
+    # Check that the basic cmap is now reversible for tuple input
+    cmap = basic_cmap((1.0, 1.0, 0.0, 1.0))
+    cmap.reversed()
```

Comment:
Since we don't actually have a dedicated test of `basic_cmap`, I suggest naming this more generally `test_basic_cmap` and testing more colour options. How about:
```python
@pytest.mark.parametrize('c', ['C0', 'k', 'gold', '#00FFFF', (1.0, 1.0, 0.0, 1.0)])
def test_basic_cmap_reversible(c):
    cmap = basic_cmap(c)

    # Check that the basic cmap is reversible
    cmap.reversed()
```